docs/readme.md

```diff
@@ -4,20 +4,27 @@
 
 Reference implementation of Kava, a blockchain for cross-chain DeFi. Built using the [cosmos-sdk](https://github.com/cosmos/cosmos-sdk).
 
+
 ## Mainnet
 
-Note, the current recommended version of the software for mainnet is v0.3.2. The master branch of this repository contains considerable development work since the last mainnet release and is **not** runnable on mainnet.
+The current recommended version of the software for mainnet is [v0.8.1](https://github.com/Kava-Labs/kava/releases/tag/v0.8.1). Note, the master branch of this repository contains development work since the last mainnet release and it may **not** be runnable on mainnet.
 
 ### Installation
 
 ```sh
-git checkout v0.3.2
+git checkout v0.8.1
 make install
 ```
 
+### Upgrade
+
+The mainnet upgrade to `kava-3` took place on June 10th, 2020 at 14:00 UTC. The version of Kava for `kava-3` is [__v0.8.1__](https://github.com/Kava-Labs/kava/releases/tag/v0.8.1).
+
+Migration instructions can be found [here](https://docs.kava.io/building/kava-3-migration-guide.html).
```

Comment:
it says that this file doesn't exist

Reply:
Could you replace it with a relative link like: `./docs/building/kava-3-migration-guide.html`?
github and the link-checker handle these automatically, but not sure about vuepress

Reply:
https://github.com/Kava-Labs/kava/pull/579/commits/accc9b33f3d47604d44120a038a90f990afa75bb